src/services/types.ts

```diff
@@ -310,6 +310,10 @@ namespace ts {
         dispose(): void;
     }
 
+    export interface GetCompletionsAtPositionOptions {
+        includeExternalModuleExports: boolean;

```

Comment:
This name suggests we're going to have a separate opt-in for each kind of completion that might return an action.  Is that the plan?

Reply:
I could not think of another feature that we would add, and thus did not have a frame of reference. We also got feedback form the VSCode team that they would like to be able to disable the feature altogether, and incidentally from [customer reports](https://github.com/Microsoft/TypeScript/issues/19764) later on.